kuka_kr16_support/launch/test_kr16_2.launch

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<launch>
+  <include file="$(find kuka_kr16_support)/launch/load_kr16_2.launch" />
+  <param name="use_gui" value="true" />
+  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher"/>
```

Comment:
Please make the `use_gui` parameter a private parameter of the `joint_state_publisher` node.